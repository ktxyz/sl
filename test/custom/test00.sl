integer age;
string name;
age := readint;  
name := readstr;
print(concatenate("Hello ", name));  
print("You are");
print(age);
print("years old");