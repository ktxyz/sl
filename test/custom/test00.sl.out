Hello Alice  
You are
25
years old